### ISSUE TITLE
PHP Connector: ZIP download (option: allowFolderDownload) requires the folder to be writable

### ISSUE THREAD

**Issue by dereks** [servocoder/RichFilemanager#130]
The PHP connector requires that a target folder be writable, in order to allow it to be downloaded as a ZIP. There is no reason for this. If I can read a dir, I should be able to read it as a convenient ZIP download. (The ZIP file itself is generated and written into /tmp/, which is writable for everyone.)

The error is in LocalFilemanager.php:874

```
		// check if file is writable
		if(!$this->has_system_permission($target_fullpath, ['w'])) {
			$this->error(sprintf($this->lang('NOT_ALLOWED_SYSTEM')));
		}
```
(I discovered this while working on the allowFolderDownload option.)

Feel free to assign this issue to me. I am in the middle of config option updates and can fix this when that is done.

**Comment by servocoder** [comment]
Nice catch. I have assigned it to you. Thanks.

**Comment by servocoder** [comment]
Fix included into the v2.4.0 release
